test_framework/terraform/aws/sles/user-data-scripts/provision_k3s_server.sh.tpl

```diff
@@ -6,6 +6,8 @@ sudo zypper install -y open-iscsi nfs-client jq
 sudo systemctl -q enable iscsid
 sudo systemctl start iscsid
 
+touch /tmp/signal
```

Comment:
I don't think we need this, but rather use cloud-init to check provisioning ready or not later.